api/cpp/include/slint_point.h

```diff
@@ -0,0 +1,22 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
```

Comment:
TBH i'm not a fan of adding one file per class like that (especially for small classes)
However, if you want to go that route, this means that we should split lots of our class  that are now in slint.h in different files

Reply:
I'm not a fan either - I'm fine with a bigger file. I saw that we have `slint_size.h` and went for this.

I could make a ticket for the next release to consolidate: Move perhaps model and window into separate files and move simple data types like size and point into slint.h. What do you think?